src/other/broadcasting.jl

```diff
@@ -35,9 +35,11 @@ end
 
 function getcolbc(bcf::Base.Broadcast.Broadcasted{Style}, colind) where {Style}
     # we assume that bcf is already flattened and unaliased
-    Base.Broadcast.Broadcasted{Style}(bcf.f,
-        map(x->Base.Broadcast.extrude(x isa AbstractDataFrame ? x[colind] : x),
-            bcf.args), bcf.axes)
+    function dfextruder(x)
```

Comment:
`map(bcf.args) do ...` is quite simpler. :-)

Reply:
fixed